{{cookiecutter.project_slug}}/{{cookiecutter.package_slug}}/__main__.py

```diff
@@ -0,0 +1,18 @@
+"""
+Entrypoint module, in case you use `python -m{{cookiecutter.project_name}}`.
+Why does this file exist, and why __main__? For more info, read:
+- https://www.python.org/dev/peps/pep-0338/
+- https://docs.python.org/2/using/cmdline.html#cmdoption-m
+- https://docs.python.org/3/using/cmdline.html#cmdoption-m
+"""
+{%- if cookiecutter.command_line_interface == 'Click' or cookiecutter.command_line_inteface == 'Argparse' %}
```

Comment:
This can be changed by {%- if cookiecutter.command_line_interface is in ["Click", "Argparse"] %} or {%- if cookiecutter.command_line_interface != 'No command-line interface' %}

Which is better?

Reply:
"IN" please :) thanks, good catch